Packs/FeedCSV/Integrations/FeedCSV/FeedCSV.yml

```diff
@@ -83,8 +83,7 @@ configuration:
   name: ignore_regex
   required: false
   type: 0
-- additionalinfo: Name of the field names in the CSV feed. If several are given, will
-    use "indicator" as the default indicator value field.
+- additionalinfo: The names to give the fields in the CSV feed. If several are given, "value" will be used as the default indicator value field.
```

Comment:
I'm not in love with this tooltip but if you feel strongly then do you. I think we need to make it _super_ clear that the field called "value" is the one passed as the value of an indicator.

Reply:
Moving to https://github.com/demisto/content/pull/5766